x-pack/plugins/security_solution/cypress/ccs_integration/detection_alerts/alerts_details.spec.ts

```diff
@@ -58,7 +58,7 @@ describe('Alert details with unmapped fields', () => {
 
   it('Displays the unmapped field on the table', () => {
     const expectedUnmmappedField = {
-      row: 56,
+      row: 89,
```

Comment:
@marshallmain In your original PR this is 88, any idea why there's a difference between master and this: https://github.com/elastic/kibana/pull/106049/files#r687374268?

Reply:
No, I saw that [this PR ](https://github.com/elastic/kibana/pull/104667/files) adjusted the row by 1 so applied the same change to mine to get it to pass. Not sure where the difference comes from.